app/src/main/res/layout/component_awesomebar.xml

```diff
@@ -6,14 +6,15 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:mozac="http://schemas.android.com/apk/res-auto"
     android:id="@+id/awesomeBar"
+    android:background="@color/photonRed50"
```

Comment:
cc @sblatz is this added background color intentional here? Or was it part of testing?